mw-webapp/cypress/fixtures/allWaysFixture.json

```diff
@@ -0,0 +1,4 @@
+{
+    "endpoint": "/ways",
+    "waysTitle": "Ways"

```

Comment:
I think it will be better to reorganize fixtures in future, but I think it is good enough for now

Reply:
Ok, let's leave it like that for now.